test/functional/test_link_change_error_inject.py

```diff
@@ -169,71 +124,51 @@ def tearDownClass(cls):
         stop_node()
 
     def test_status_code_error_inject(self):
-        status_code_error1 = {
+        cmd_nic_down = {
             "execute": "set_link",
             "arguments": {
-                "name": "e1000.0",
+                "name": "e1000.1",
                 "up": False
             }
         }
-        status_code_error2 = {
+        cmd_nic_up = {
             "execute": "set_link",
             "arguments": {
-                "name": "e1000.0",
+                "name": "e1000.1",
```

Comment:
what is the difference between e1000.0 and e1000.1?

Reply:
e1000.1 is the nic which i want to operate.
e1000.0 is the nic provides ssh tunnel.

Reply:
ok.